### PR TITLE
Scaled Icons for Smaller Screen Sizes + Disappearing Icons (1300px)

### PR DESCRIPTION
**Pull request by TheHiddenLayer**

### CODE REVIEW

**Review by Anshul-Birla — DISMISSED** [review]
sweet looks like it should work

File: src/css/SocialFollow.css [hunk 1]
```diff
@@ -63,13 +64,31 @@
...
         position: relative;
         display: inline-block;
         width: 120px;
         border-radius: 2%;
+        line-height: 130%;
     }
+}
 
+/* MAKE ICONS DISAPPEAR */
+@media screen and (max-width:1300px) {
```
Comment (Anshul-Birla) [inline src/css/SocialFollow.css]: UPO CHANGE THIS should be 600 px it needs to disappear for mobile not desktop

**Review by Anshul-Birla — COMMENTED** [review]
What is said below

**Review by jayrc7 — DISMISSED** [review]
sweet thanks Upo and Anshul!

File: src/css/SocialFollow.css (hunk 1 above)
Comment (TheHiddenLayer) [reply src/css/SocialFollow.css]: Yep it's changed

**Review by Anshul-Birla — APPROVED** [review]
looks good

**Review by jayrc7 — APPROVED** [review]
Nice nice

**Pull request merged**